packages/models-library/src/models_library/clusters.py

```diff
@@ -110,11 +110,23 @@ class Config:
 
 
 class Cluster(BaseCluster):
-    id: ClusterID = Field(..., description="The cluster ID")
+    id: Union[ClusterID, Literal["default"]] = Field(..., description="The cluster ID")
 
     class Config(BaseCluster.Config):
         schema_extra = {
             "examples": [
+                {
+                    "id": "default",
+                    "name": "The default cluster",
+                    "type": ClusterType.ON_PREMISE,
```

Comment:
I would be curious what the choice of on_premise VS aws implies (if you didnt answer yet in the last PRs review ;) )

Reply:
yes, currently this field is not really useful. I am on the verge of removing it.